docs/assets/tables/all_outputs.tsv

```diff
@@ -192,6 +192,15 @@ checkv_denovo_total_genes	Int	Number of genes detected in de novo assembly by Ch
 checkv_denovo_version	String	Version of CheckV used for de novo assembly	TheiaViral_Illumina_PE, TheiaViral_ONT
 checkv_denovo_weighted_completeness	Float	Weighted completeness score for de novo assembly from CheckV	TheiaViral_Illumina_PE, TheiaViral_ONT
 checkv_denovo_weighted_contamination	Float	Weighted contamination score for de novo assembly from CheckV	TheiaViral_Illumina_PE, TheiaViral_ONT
+chroquetas_amr_stats	File	File depicting quantity of AMR mutations detected on a gene-by-gene basis	TheiaEuk_Illumina_PE, TheiaEuk_ONT
+chroquetas_amr_stats_file	File	File depicting quantity of AMR mutations detected on a gene-by-gene basis	ChroQueTas
+chroquetas_amr_summary	File	File depicting positions, mutations, and associated fungicide resistance	TheiaEuk_Illumina_PE, TheiaEuk_ONT
+chroquetas_amr_summary_file	File	File depicting positions, mutations, and associated fungicide resistance	ChroQueTas
```

Comment:
i would recommend removing the _file output for the ChroQueTas standalone because this seems quite redundant